### PR TITLE
Microverse Hello Project Pull Request

### PR DESCRIPTION
**Pull request by ernestkorir**
Hello Project

### CODE REVIEW

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @ernestkorir, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  Correct GitHub Flow. :heavy_check_mark: 
 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Good job so far @ernestkorir, you've put a ton of effort to push your project this far. However, there're few things you need to work on before moving to the next project.
Kindly set up your linter correctly so that it can run tests on your pull request. You can find how to set up linters for html and css [here](https://github.com/microverseinc/linters-config/tree/master/html-css). Kindly take your time  to set it up properly.

- [x] Your README file looks too scanty. Kindly follow this [template](https://github.com/microverseinc/readme-template) from Microverse to make a professional README.
- [ ] It is required and good practice to give your pull request a descriptive title, brief summary of what you've done in that pull request. In your own case, your PR lacks the PR summary. Kindly look into that and fix it.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>
```
Comment (Qoosim) [inline index.html]: - [ ] Kindly give a class name to your `h1` element and use that to style the content of the page.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: aqua;
+}
```
Comment (Qoosim) [inline style.css]: - [ ] Kindly use the class name from your index file to style the content of the `h1`.

**Review by Qoosim — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (ernestkorir) [reply style.css]: Hi @Qoosim recommended changes have been made. 

Github workflows fixed - there was a typo in the workflows directory.

Thanks,
E.

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @ernestkorir , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [ ] Linter checks are passing
- [ ] Application is working fine
- [ ] Descriptive commit messages

## Optional suggestions
[ Optional ] please consider updating this section of your [readme file](https://github.com/ernestkorir/HelloProject), filling this section of the [readme](https://github.com/ernestkorir/HelloProject) will give users who want to set up your project locally on their computers an easy time

Screenshot from 2022-09-21 01-16-02

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**